apiserver/plane/app/views/asset/v2.py

```diff
@@ -718,3 +718,97 @@ def get(self, request, slug, asset_id):
             id=asset_id, workspace__slug=slug, deleted_at__isnull=True
         ).exists()
         return Response({"exists": asset}, status=status.HTTP_200_OK)
+
+
+class DuplicateAssetEndpoint(BaseAPIView):
+
+    throttle_classes = [ScopedRateThrottle]
```

Comment:
@NarayanBavisetti Create a custom throttle class that rate limits based on the asset id